src/app/admin/components/microschema-list/microschema-list.component.html

```diff
@@ -1,18 +1,41 @@
-<div class="not-found"
-     *ngIf="(microschemas$ | async).length === 0">
-    <div>
-        <icon>sentiment_very_dissatisfied</icon>
+<mesh-admin-list [items]="microschemas$ | async"
+                 [itemsPerPage]="2"
```

Comment:
Change to 10

Reply:
Done for schema-list.component.html and microschema-list.component.html.